scripts/eurostat/birth_death_migration/test/test1/data.tsv

```diff
@@ -0,0 +1,300 @@
+indic_de,geo\time	2019 	2018 	2017 	2016 	2015 	2014 	2013 	2012 	2011 	2010 	2009 	2008 	2007 	2006 	2005 	2004 	2003 	2002 	2001 	2000 
```

Comment:
Do we need 300 rows? For testing, it should be doable by hand. Please select a few but good cases (look for edge cases or variety).